docs/Config.md

```diff
@@ -469,6 +469,7 @@ gui:
 
 ![border example](../../assets/colored-border-example.png)
 
+You can find preset themes on [this repository](https://github.com/catppuccin/lazygit) by Catppuccin.
```

Comment:
```suggestion
You can find preset themes on [this repository](https://github.com/catppuccin/lazygit) by Catppuccin (this is unofficial: be sure to check the source!)
```

Adding a warning here just to cover my ass in case some malicious person goes and adds a custom command to one of those config files which steals user data or something.